src/utils/mailClient.js

```diff
@@ -0,0 +1,21 @@
+import { NotifyClient } from 'notifications-node-client'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+class NotifyClientSingleton {
```

Comment:
I'd leave it to the language to ensure there's only one instance by defining `export const client = new NotifyClient(...)`. Less code and less things to test.